nivio-demo/docker-compose.yml

```diff
@@ -11,7 +11,9 @@ services:
 
   nivio:
     image: dedica/nivio:latest
-    #environment:
+    environment:
+      SEED: ${SEED}
+      DEMO: ${DEMO}
       # enable DEMO to have a preloaded landscape
       # DEMO: 1
       # SEED also loads data, but uses the sources from the webserver
```

Comment:
To make the examples in the manual work right out of the box, we should declare the environment variables. If they are not provided, they default to an empty string. This should be no problem for our backend, right?

Reply:
If neither SEED nor DEMO is given, nivio starts without loading any configuration. Theoretically one could push a config via rest api, but I'm not sure this is still working. Empty strings should be equal to absent vars. So yes, we should give examples.